query-engine/connectors/sql-query-connector/Cargo.toml

```diff
@@ -45,6 +38,9 @@ opentelemetry = { version = "0.17", features = ["tokio"] }
 tracing-opentelemetry = "0.17.3"
 cuid = { git = "https://github.com/prisma/cuid-rust", branch = "wasm32-support" }
 quaint.workspace = true
+# quaint = { path = "../../../quaint", default-features = false, features = [
```

Comment:
@SevInf Is this correct?

Reply:
Sorry if it wasn't clear but this comment should be removed